os/net/ipv6/uipopt.h

```diff
@@ -518,6 +518,35 @@ void uip_log(char *msg);
 
 #define UIP_DEFAULT_PREFIX_LEN 64
 
+/**
+ * Enables selection of maximal MAC-layer transmission count at application layer
+ */
+#ifdef UIP_CONF_WITH_VARIABLE_RETRANSMISSIONS
+#define UIP_WITH_VARIABLE_RETRANSMISSIONS UIP_CONF_WITH_VARIABLE_RETRANSMISSIONS
+#else
+#define UIP_WITH_VARIABLE_RETRANSMISSIONS 0
+#endif
```

Comment:
do we really need to make this optional?

Reply:
`.rom` overflow on Sky otherwise.

Reply:
argh, that is going to be annoying soon enough :s